packages/metadata/src/__tests__/unit/decorator-factory.unit.ts

```diff
@@ -352,7 +352,7 @@ describe('PropertyDecoratorFactory', () => {
 
   it('throws if applied more than once on the same property', () => {
     expect(() => {
-      // tslint:disable-next-line:no-unused
+      /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
```

Comment:
```suggestion
      // eslint-disable-next-line @typescript-eslint/no-unused-vars */
```

Reply:
It makes me wonder - if we removed the class name, would eslint still complain about unused variables?

```ts
expect(() => {
  class {
    @propertyDecorator({x: 1})
    @propertyDecorator({y: 2})
    myProp: string;
  }
}).to.throw(
  // ...
);
```

Reply:
A class declaration without `default` modifier must have a name.